source/iostreams.tex

```diff
@@ -11259,10 +11259,10 @@
 be the same as one \grammarterm{directory-separator} character.
 
 \pnum
-The filename \grammarterm{dot} is treated as a reference to the current directory.
-The filename \grammarterm{dot-dot} is treated as a reference to the parent directory.
-What the filename \grammarterm{dot-dot} refers to
-relative to \grammarterm{root-directory} is \impldef{meaning of \grammarterm{dot-dot} in \grammarterm{root-directory}}.
+The filename dot is treated as a reference to the current directory.
```

Comment:
"The filename dot" sounds like a particular kind of dot, not a particular kind of filename. How about changing to "The dot filename" and "the dot-dot filename" throughout?

Reply:
It turns out only this place needs changing; all other places already say "dot filename" or similar.